Automattic/woocommerce-payments
# Remove the color fields and icon on receipt settings page

Fixes #4144 

#### Changes proposed in this Pull Request
Removal of Stripe specific branding options:

- Color fields
- Icon upload component

I also adjusted the copy to reflect that logo will be used only for printed receipts.









**Before**
-------------
<img width="591" alt="Screen Shot 2022-04-22 at 17 20 31" src="https://user-images.githubusercontent.com/1553182/164744715-38011eb7-68be-491c-81ce-64c6eb3c4ed6.png">

**After**
-------------
<img width="591" alt="Screen Shot 2022-04-22 at 17 17 27" src="https://user-images.githubusercontent.com/1553182/164744171-c8f9f736-789d-4976-8b50-d8d1918a5134.png">

#### Testing instructions





* Go to Payments > Card Readers > Receipt Details
* Check that the icon upload and color picker controllers are no longer present
* Check that the text below Branding section is updated.
* Check that settings can be updated/saved without errors

-------------------


**Post merge**



- [x] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : [Testing instructions](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions-for-WC-Payments-4.1.0#remove-the-color-fields-and-icon-on-receipt-settings-page)
